drop-analytics/src/lib.rs

```diff
@@ -1,20 +1,28 @@
 #[cfg(feature = "moose")]
 mod moose_impl;
 
+mod file_impl;
```

Comment:
I mean here `#[cfg(feature = "moose_file")]`

Reply:
the file implementation, just like the mock one, doesn't use actual moose, it's a self-contained implementation, don't see why it shouldn't compile here

Reply:
Oh yeah, you're right. It should compile.
Nonetheless, I still think it's better to gate the implementation under the feature flag so that the code is not even present in the production build